script/script-v0.0.1.schema.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
we should also have a key for schema version if we are to version our json files. 

If i understand user would have to do 
```
schemaver: 0.1
slurm_check:
  description: "description of test"
  shell: "sinfo"

maintainers:
  - "First Last <first.last@example.com>"
```

Reply:
Are you saying there is an issue with just having `version` ?